### PR TITLE
[frontend] Fix correlated cases is not working

### LINKED ISSUE #4803: Correlated cases is not working
## Description

As an example (working):

1. Create Report 1 and add Observable A in the report
2. Create Report 2 and add Observable A in the report

In both report, "Correlated reports" display the other one.

Do the same thing with Case 1 and Case 2.

=> Nothing in Correlated cases.

### PR DESCRIPTION
**Pull request by Megafredo**
### Proposed changes

* Updating the ternary operator in the "CaseIncidentDetail" file
* Adding the ternary operator in the "ReportDetail" file to replace empty space with '-'

### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/4803

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/cases/case_incidents/CaseIncidentDetails.tsx [hunk 1]
```diff
@@ -233,7 +233,7 @@ const CaseIncidentDetails: FunctionComponent<CaseIncidentDetailsProps> = ({
           {t('Correlated cases')}
         </Typography>
         <List classes={{ root: classes.relatedContainers }}>
-          {expandable
+          {(data.relatedContainers?.edges ?? []).length >= 2
```
Comment (jpkha) [inline opencti-platform/opencti-front/src/private/components/cases/case_incidents/CaseIncidentDetails.tsx]: I think you can write something like this instead :
`data.relatedContainers?.edges?.length >= 2`

File: opencti-platform/opencti-front/src/private/components/analyses/reports/ReportDetails.jsx [hunk 2]
```diff
@@ -168,7 +168,8 @@ const ReportDetailsComponent = (props) => {
           {t('Correlated reports')}
         </Typography>
         <List>
-          {R.take(expanded ? 200 : 5, report.relatedContainers.edges)
+          {(report.relatedContainers?.edges ?? []).length >= 2
```
Comment (jpkha) [inline opencti-platform/opencti-front/src/private/components/analyses/reports/ReportDetails.jsx]: I think you can write something like this instead :
data.relatedContainers?.edges?.length >= 2

**Review by jpkha — APPROVED** [review]

File: opencti-platform/opencti-front/src/private/components/cases/case_incidents/CaseIncidentDetails.tsx [hunk 3]
```diff
@@ -233,7 +233,7 @@ const CaseIncidentDetails: FunctionComponent<CaseIncidentDetailsProps> = ({
           {t('Correlated cases')}
         </Typography>
         <List classes={{ root: classes.relatedContainers }}>
-          {expandable
+          {(data.relatedContainers?.edges && data.relatedContainers.edges.length >= 2)
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/cases/case_incidents/CaseIncidentDetails.tsx]: Not sure, let's talk about that

**Review by Kedae — CHANGES_REQUESTED** [review]

File: opencti-platform/opencti-front/src/private/components/cases/case_incidents/CaseIncidentDetails.tsx (hunk 3 above)
Comment (Kedae) [reply opencti-platform/opencti-front/src/private/components/cases/case_incidents/CaseIncidentDetails.tsx]: You need to extract the filtering of the list before the return and check that this computed list as at least one element

**Pull request merged**